guide/misc/download.md

```diff
@@ -159,16 +159,16 @@ You can download archives of the source directly:
   <tr>
     <td style="vertical-align: middle;"><center>{{ site.brooklyn.git_branch }}</center></td>
     <td>
-<a href="https://github.com/apache/incubator-brooklyn/tarball/{{ site.brooklyn.git_branch }}"><img border="0" width="90" src="https://github.com/images/modules/download/tar.png"></a>
-<a href="https://github.com/apache/incubator-brooklyn/zipball/{{ site.brooklyn.git_branch }}"><img border="0" width="90" src="https://github.com/images/modules/download/zip.png"></a>
+<a href="https://github.com/apache/brooklyn/tarball/{{ site.brooklyn.git_branch }}"><img border="0" width="90" src="https://github.com/images/modules/download/tar.png"></a>
+<a href="https://github.com/apache/brooklyn/zipball/{{ site.brooklyn.git_branch }}"><img border="0" width="90" src="https://github.com/images/modules/download/zip.png"></a>
```

Comment:
probably not very useful for people if they get the zip source of the root project.  they won't have submodules.  probably remove this section, make people use git.  or point people at the official apache source release.